### ISSUE TITLE
The rspec file is missing a few tests.

### ISSUE THREAD

**Issue by Sdcrouse** [learn-co-curriculum/html-album-cover#27]
Hi. In the index.html file, one of the comments in the `<div id="front" class="album">` element says:

`<!-- Level 4 '| FROM DESPICABLE ME...' Header -->`

The first test (in the `context 'within <body>'` section) should check for that, but it does not; it only checks for the first `<h4>` element and the `<h2>` element. Effectively, doing this will pass the tests:

```
<div id="front" class="album">
    <div id="title-box">
    <h4>PHARRELL WILLIAMS</h4>
    <h2>HAPPY</h2>
</div>
```

But, it's supposed to look like this:

```
<div id="front" class="album">
    <div id="title-box">
    <h4>PHARRELL WILLIAMS</h4>
    <h2>HAPPY</h2>
    <h4>| FROM DESPICABLE ME 2 |</h4>
</div>
```

I also found an issue regarding the second test ( `the back section contains...` ). It checks to make sure that there are two `<h3>` tags in the "back" section, but it doesn't check their content. Basically, the following code will pass the test:

```
<div id="back" class="album">
    <h3></h3>
    <h1>HAPPY</h1>
    <h3></h3>        
    <!-- The rest of the code (not relevant to this issue) -->
</div>
```

But it should look like something like this:

```
<div id="back" class="album">
    <h3>| &nbsp;&nbsp; FROM DESPICABLE ME 2 &nbsp;&nbsp; |</h3>
    <h1>HAPPY</h1>
    <h3>WRITTEN BY PHARRELL WILLIAMS</h3>
    <!-- The rest of the code (again, not relevant to this issue) -->
</div>
```

Lastly, near the bottom of index.html is this comment:

`<!-- span text 'COLUMBIARECORDS.COM' -->`

However, none of the tests look for that specific `<span>` element.

As always, thanks for looking into this!

Sdcrouse

**Comment by maxwellbenton** [comment]
Hi @Sdcrouse,

Thanks for the thorough feedback on this lab. I've added tests so that this lab now tests for all of the HTML elements students are expected to create. I've also rearranged the tests and improved them so students can follow them to build the exact lay out of the album cover.

Thank you, as always, for providing feedback. You've done a lot to help us polish many learn.co materials!
